### ISSUE TITLE
Verifying the "aud" claim of a WPT

### ISSUE THREAD

**Issue by yaronf** [ietf-wg-wimse/draft-ietf-wimse-s2s-protocol#26]
Current text: "The `aud` claim of the WPT matches the target URI, or an acceptable alias or normalization thereof, of the HTTP request".

@yaronf: Man, what's a poor implementer to do with "an acceptable alias or normalization thereof"?

@bc-pi: Whatever they need to do to get shit to work in the case there are proxies in the middle that rewrite the path or even host. I agree that the language is less than ideal but think some allowance is needed.

https://github.com/yaronf/wimse-s2s/pull/19/files#r1648137394 for more context

**Comment by arndt-s** [comment]
Do we need audience if we have a good proof of possession coverage via other claims? 🤔

**Comment by bc-pi** [comment]
> Do we need audience if we have a good proof of possession coverage via other claims? 🤔

Audience is, to the best of my always limited understanding, an integral part of that proof of possession coverage.

**Comment by jsalowey** [comment]
Yes, omitting the audience would degrade the security of the WIT and WPT.   Perhaps we can say something like:

"The audience SHOULD match the target URI.  However, there may be some normalization, rewriting or other process that requires the audience to be set to a different value that is defined by local policy. "

**Comment by yaronf** [comment]
I can live with this language.
